### ISSUE TITLE
What datasets pretrained weights trained on?

### ISSUE THREAD

**Issue by flying318** [ChenHongruixuan/MambaCD#42]
Hi, I wonder which data you used to pretrain the weights: [VMamba-Tiny](https://github.com/MzeroMiko/VMamba/releases/download/%2320240316/vssm_tiny_0230_ckpt_epoch_262.pth), [VMamba-Small](https://github.com/MzeroMiko/VMamba/releases/download/%2320240316/vssm_small_0229_ckpt_epoch_222.pth), and [VMamba-Base](https://github.com/MzeroMiko/VMamba/releases/download/%2320240316/vssm_base_0229_ckpt_epoch_237.pth)? I download the original weights of [Vmamba](https://github.com/MzeroMiko/VMamba/releases/download/%2320240223/vssmsmall_upernet_4xb4-160k_ade20k-512x512_iter_160000.pth)(same version of `Mamba_Small`) to train and get the relative lower accuracy on same dataset, compared to pretrained weights you provided. 😞 
I want to generate new pretrained weights on a new mamba and I will be appreciated if you introduce it. 😃

**Comment by ChenHongruixuan** [comment]
Hi,

Thank you for your question. 

> Hi, I wonder which data you used to pretrain the weights: 
We used all three weights. And there is a one-to-one correspondence with VMamba, e.g. MambaBCD-Tiny uses VMamba-Tiny.

> I want to generate new pretrained weights on a new mamba and I will be appreciated if you introduce it.
I don't really understand what you mean by this, could you explain it in detail?

**Comment by flying318** [comment]
Hi,
Thanks for your reply. I just have a question about how the pretrained weights of `Vmamba-Tiny` and other two versions are generated, for I want to get new weights in same way if you could introduce details. 😀

**Comment by ChenHongruixuan** [comment]
Hi,

What do you mean of [I want to get new weights in same way]? If you're asking about the pre-trained weights of the encoder (VMamba), we didn't pre-train it ourselves but directly used the original weights of VMamba pretrained on ImageNet.

Best,

**Comment by flying318** [comment]
Hi,
I think I get your answer. Thank you again.
